### PR TITLE
added reaper_medium::Reaper::time_map_2_qn_to_measure

### PR DESCRIPTION
**Pull request by Levitanus**
also introduced the new struct `TimeMapQNToMeasuresResult` for storing bar start\end positions

### CODE REVIEW

**Comment by Levitanus** [comment]
P.S. Let's start from small pull. I hope to push medium API a bit forward next month)

File: main/medium/src/reaper.rs [hunk 1]
```diff
@@ -6972,6 +7020,16 @@ pub struct TimeMapGetMeasureInfoResult {
     pub tempo: Bpm,
 }
 
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub struct TimeMapQNToMeasuresResult {
```
Comment (helgoboss) [inline main/medium/src/reaper.rs:7024]: I follow Rust API conventions when it comes to acronyms (https://rust-lang.github.io/api-guidelines/naming.html).

```suggestion
pub struct TimeMapQnToMeasuresResult {
```

**Comment by Levitanus** [comment]
Suggestions are applied

**Comment by helgoboss** [comment]
@Levitanus Now the only thing left is running `cargo fmt`.

**Comment by helgoboss** [comment]
I shouldn't have merged this, it was not building ;) Never mind. Fixed it on master.

**Pull request merged**